src/Nowruz/modules/Search/components/FilterSlider/index.tsx

```diff
@@ -45,6 +45,7 @@ export const FilterSlider: FC<FilterSliderProps> = ({ type, onApply, onClose, fi
           value={filters.location}
           loadOptions={searchCities}
           defaultOptions
+          creatable
```

Comment:
This property shouldn't be set for location. We use this prop for some entities like 'Organization' which user can create a new one. But user is not allowed to create a location
![image](https://github.com/socious-io/web-app-v2/assets/60564928/d8c15d30-5c91-4b38-b730-a92c4d116603)

Reply:
it was a typo, fixed thanks